src/idl_parser.cpp

```diff
@@ -1651,7 +1652,13 @@ CheckedError Parser::ParseSingleValue(const std::string *name, Value &e,
 #undef TRY_ECHECK
 #undef IF_ECHECK_
 
-  if (!match) return TokenError();
+  if (!match) {
+    std::string msg;
+    msg += "Cannot assign token starting with '" + TokenToStringId(token_) +
+           "' to value of <" + std::string(kTypeNames[in_type]) + "> type.";
+    msg += " Please, check the type name and namespace.";
```

Comment:
This last line is probably superfluous, we don't urge users to go check things in any other errors.

Reply:
https://github.com/google/flatbuffers/blob/9fa8245e81a2fd8ad19f9ae8f3da3a00e796e462/src/idl_parser.cpp#L2765-L2766

"check namespace"

Reply:
Ok :)
Though generally don't think errors should have these.